sql/updates/latest-dev.sql

```diff
@@ -1,6 +1,7 @@
 -- Add new function definitions, columns and tables for distributed hypertables
 DROP FUNCTION IF EXISTS create_hypertable(regclass,name,name,integer,name,name,anyelement,boolean,boolean,regproc,boolean,text,regproc,regproc);
 DROP FUNCTION IF EXISTS add_drop_chunks_policy;
+DROP FUNCTION IF EXISTS add_retention_policy;
```

Comment:
thats wrong, you are adding this function, but you need to drop remove_drop_chunks_policy